admin/tabs/cluster/ClusterTab.ts

```diff
@@ -31,19 +28,39 @@ export const clusterTab = hoistCmp.factory({
                 },
                 item: grid()
             }),
-            instance?.isReady
-                ? panel({
-                      compactHeader: true,
-                      tbar: [
-                          box({width: 150, item: model.formatInstance(instance)}),
-                          hspacer(25),
-                          tabSwitcher()
-                      ],
-                      flex: 1,
-                      item: tabContainer()
-                  })
-                : placeholder(Icon.server(), 'Select a running instance above.'),
-            mask({bind: model.loadModel})
+            detailPanel(),
+            mask({
```

Comment:
Could split the mask out into its own component if we want to minimize re-renders of the top-level `clusterTab` when `lastLoadException` changes